### ISSUE TITLE
shape '[-1, 200, 18]' is invalid for input of size

### ISSUE THREAD

**Issue by liyc-ai** [PKU-AI-Edge/CORRO#3]
Hi, thanks to your work.

I cloned CORRO and ran the commands as instructed in README, with the env type "walker_param". The **Data Collection** ran well. But when I tried to pretrain the CVAE, error "shape '[-1, 200, 18]' is invalid for input of size" occurred at line 179 of `offline_utils.py`. It seems that the num of transitions collected can not be divided by `200*18`. Maybe it is due to that the trajectories collected not always have length 200. What should I do? Can I remove all the trajectories with length less than 200?

**Comment by Folly135** [comment]
Hello, I would like to know if you have resolved this issue?

**Comment by liyc-ai** [comment]
No, I carefully checked the code and found that it is due to that not all sampled trajectories have the same length, which may happen in environments like walker. I personally think that the current code does not deal with this issue. A naive approach is to pad all the trajectories to the same length.
